controllers/record.go

```diff
@@ -42,6 +42,7 @@ func (c *ApiController) GetRecords() {
 	value := c.Input().Get("value")
 	sortField := c.Input().Get("sortField")
 	sortOrder := c.Input().Get("sortOrder")
+	organizationName := c.Input().Get("organizationName")
```

Comment:
What u mean?  method GetRecords at backend returns all records for global admin and filters by user.Owner for other. So how we can send information about selected organization by global admin without adding new param?

Reply:
Do you know org admin in Casdoor?

@ISulimanov do you get what I said? Maybe you can explain my idea if yes

Reply:
> Do you know org admin in Casdoor?
> 
> 
> 
> @ISulimanov do you get what I said? Maybe you can explain my idea if yes

So if I got you point - 
Org admin is able to see only own organisations entities, 
We have to reuse same approach when we select org in context menu.
Is it?

Reply:
@ISulimanov yes, re-use the existing org-admin code as much as possible. The backend API code is already so complex as you have seen. We need to avoid adding more dimensions if possible

Reply:
@hsluoyz fully agree that reusing existing code path usually better rather than adding new one, 
there is only one thing that bother me in current implementation - https://github.com/casdoor/casdoor/issues/2022

Reply:
@ISulimanov this bug has been fixed

Reply:
and this fix created new bug: https://github.com/casdoor/casdoor/issues/2024
We need new parameter "organizationName" at GetOrganizations for current process cases when global admin want to get only 1 organization. Same for GetRecords/GetChats

Reply:
@hsluoyz 

We re-use the existing org-admin code as much as possible, 
but still there are 2 cases:

### Organization tab: 

Not clear how to filter org for global admin:
Three options: 

- Don't make filter at all 
- Use field/value params for filter. Just workaround because leads to bug when user will try to make search by another column 
- Keep new param like orgName

Records tab: 

Since org params retrieving from backend side like:

```
organization, ok := c.RequireAdmin() 
....
return user.Owner, true
```
Looks like we have to add new get params for make it works

Reply:
OK